apps/test/package.json

```diff
@@ -1,18 +1,5 @@
 {
   "name": "@cdo/apps/test",
   "version": "0.0.1",
-  "description": "Tests for cdo apps",
-  "browserify": {
-    "transform": [
-      ["babelify", {"compact": false, "ignore": "blockly"}],
-      "require-globify",
-      "browserify-shim"
-    ]
-  },
-  "browserify-shim": {
-    "blockly": {
-      "exports": "Blockly"
-    },
-    "marked": "global:marked"
-  }
+  "description": "Tests for cdo apps"
 }
```

Comment:
Possible that we don't even need this package.json now that we don't need to do browserify stuff in here?